packages/fuels/tests/logs/script_needs_custom_decoder_logging/src/main.sw

```diff
@@ -0,0 +1,6 @@
+script;
+
+fn main() {
```

Comment:
Depending on the final API, it might well be possible to remove it. We'll have to wait for it to land. I'd rather leave a TODO here than make a small tech-dept issue block the PR.